lib/bolt/analytics.rb

```diff
@@ -0,0 +1,143 @@
+# frozen_string_literal: true
+
+require 'bolt/version'
+require 'httpclient'
+require 'json'
+require 'locale'
+require 'logging'
+require 'securerandom'
+
+module Bolt
+  module Analytics
+    PROTOCOL_VERSION = 1
+    APPLICATION_NAME = 'bolt'
+    TRACKING_ID = 'UA-fakefakefake-1'
```

Comment:
Still waiting on a TRACKING_ID?

Reply:
Yeah. The ticket has been set "in progress", so I'm hopeful.